test/widget/auth/auth_page_test.dart

```diff
@@ -2,8 +2,6 @@ import 'package:crowdleague/actions/auth/sign_in_with_apple.dart';
 import 'package:crowdleague/actions/auth/sign_in_with_google.dart';
```

Comment:
The rest of the PR is just replacing boilerplate setup code with using the test harnesses.  I won't attempt to go through and add github comments to everything because there's quite a bit of it and it'll be super repetitive.  Whoever is reviewing I would say feel free to take my word for it about what the changes are and rely on the fact that the tests all run successfully rather than reading through everything... your call of course!  And of course, don't hesitate to ask me to explain any of it.  Thanks! 🙂